src/feature/featurelist.h

```diff
@@ -222,3 +222,10 @@ FEATURE(webPurchase,           // Feature ID
         FeatureCallback_true,  // Can be flipped off
         QStringList(),         // feature dependencies
         FeatureCallback_webPurchase)
+
+FEATURE(localProxy,                      // Feature ID
+        "LocalProxy",                    // Feature name
+        FeatureCallback_proxyCanTurnOn,  // Can be flipped on
+        FeatureCallback_true,            // Can be flipped off
+        QStringList(),                   // feature dependencies
```

Comment:
Shouldn't this have a dependency on split tunnelling?

Reply:
true!